crates/but/src/command/legacy/rub/mod.rs

```diff
@@ -298,71 +310,33 @@ fn parse_range(ctx: &mut Context, id_map: &IdMap, source: &str) -> anyhow::Resul
     ))
 }
 fn get_all_files_in_display_order(ctx: &mut Context, id_map: &IdMap) -> anyhow::Result<Vec<CliId>> {
```

Comment:
I didn't realise this before, but my guess here is that this will bite as soon as it can, unless I am getting this wrong in thinking that `but rub` is coupled with `but status` because of the range support where display order matters.